dev_guide/templates.adoc

```diff
@@ -212,3 +212,8 @@ database name                                root
 
 The output identifies several parameters that are generated with a regex
 expression generator when the template is processed.
+
+== Using the QuickStart Templates
+OpenShift provides a number of templates in the box to make it easy to quickly get started creating a new application for different languages.  Templates are provided for Rails(Ruby), Django(Python), NodeJS, CakePHP(PHP), and Dancer(Perl).  Your administrator should have registered these templates in the `openshift` project so you have access to them.  If not, direct him or her to the link:../admin_guide/install/first_steps.html[first steps] guide.
```

Comment:
@rhcarvalho @sspeiche @jhadvig @ewolinetz this is how i suggest we handle explaining to users how to use the templates.  You can link to this doc/section from the welcome page if you want, but let's not provide all this material in every repo.

(and we can improve this section of course)